test.js

```diff
@@ -51,40 +51,6 @@ test('stdout/stderr/all available on errors', async t => {
 	t.is(typeof all, 'string');
 });
 
-test('stdout/stderr/all are undefined if ignored', async t => {
```

Comment:
You accidentally removed all those tests. Just copy/paste them again and commit them, it should fix the diff.

Reply:
Oops, now they are back: c092016523a8c4d9f7ac19e95fc89fd0d633dc11